### DISCUSSION TITLE
game-dev/unity/vscode-integration/

### DISCUSSION THREAD

**Discussion by giscus** [tedliou/blog-discussion#3] [General]
# game-dev/unity/vscode-integration/

Visual Studio 和 VS Code 是 Unity 開發者最常使用的 C# 腳本編輯工具，通常都建議用 Visual Studio，但因為它軟體比較臃腫，導致有些電腦跑不太動，這時可以考慮改用輕量化的 VS Code 來開發遊戲！在 Unity 6 的開發環境中，我們需要在 Unity 中安裝 VS Code 支援、在 VS Code 安裝 C# 的支援，才能正常顯示自動完成選單，來高效率的的用 VS Code 來撰寫腳本。

https://vervecode.dev/game-dev/unity/vscode-integration/

**Comment by tedliou** [comment]
Rider 現在開放非商用免費授權，個人開發者都可以用！同時支援 Windows、macOS 和 Linux 平台，也是一個很不錯的工具！
官網：https://www.jetbrains.com/rider/
